### DISCUSSION TITLE
trying to connect to my raspberry pi using Zimodem 3.7 No carrier issue.

### DISCUSSION THREAD

**Discussion by johngalt01** [bozimmerman/Zimodem#152] [Q&A] [answered]
Hello,

the issue i have: when i ATDT into my raspberry pi, i get presented with login: but if i type anything other then 'enter' my pi instantly kicks me off with a NO Carrier. 

if i use my Oldnet wifi modem i can connect login and stay connected.

i'm not sure what setting i need to change on the ZIMODEM. otherwise so far i like it.

please help :(

**Comment by johngalt01** [comment]
this is interesting

if i use my windows 10 machine and teraterm then the connection goes through... but if it use my old computer modem software using IMP 2.45 then it disconnects.

**Reply by sswazey** [reply, marked as answer]
Check the DTR pin and DTR LED.  The DTR LED should be on whenever the host terminal program is running.  If it toggles from on to off it will hang up the line.  You should also check the cable on your old computer to confirm it is wired correctly.  Also try using a 3-wire cable from the modem to the old computer that connects only pins 2,3, and 7.   

For a quick fix try the following command:

**ATS54=2**

this will force DTR on and disable the hangup.

**Comment by johngalt01** [comment]
Hi i just saw your response and i wanted to let you know that ATS54=2 solved the issue.
